test/getPublicKey.spec.js

```diff
@@ -1,116 +1,31 @@
 'use strict'
 
-const nock = require('nock')
```

Comment:
removed most of the tests here because this function is simply doing a conversion on the result of the getJwk function, so it doesn't need to be tested apart from the interactions with the other function